mongo/mongo.go

```diff
@@ -713,7 +713,7 @@ func packagesForSeries(series string) ([]string, []string) {
 	switch series {
 	case "precise", "quantal", "raring", "saucy", "centos7":
 		return []string{"mongodb-server"}, []string{}
-	case "trusty", "wily", "xenial":
+	case "trusty", "wily":
 		return []string{JujuMongoPackage, JujuMongoToolsPackage}, []string{"juju-mongodb"}
```

Comment:
Why not get rid of all unsupported (in any way) versions? We're still missing utopic and vivid here so IMHO keeping quantal, raring and saucy makes no sense..

Reply:
I talked about it here, and Ian agreed, also we felt like guessing on Trusty was just asking for something to go wrong in the future. So its been updated to stop guessing.